lms/static/scripts/postmessage_json_rpc/server.js

```diff
@@ -6,6 +6,8 @@
  * @prop {string} method
  * @prop {any[]} params
  * @prop {string|null} id
+ *
+ * @typedef {Omit<JsonRpcRequest, 'id'>} JsonRpcNotificationRequest
```

Comment:
I've attempted to adhere to the (slightly-eccentric) casing as established in this module throughout.

Reply:
Perhaps shorten the type name to `JsonRpcNotification`, as it isn't really a "request" if no response is expected.